buddyboss/buddyboss-platform
# PROD-948 Fix - Group message email notification is not sending when the organizer has an Instructor role

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #937

### How to test the changes in this Pull Request:

1. Make sure that you have BuddyBoss Platform, LearnDash LMS, and Instructor Role plugins installed and activated.
2. Enable BuddyBoss > Settings > Groups > Group Messages.
3. Create a user with Instructor Role.
4. In the BuddyBoss platform, create a Social Group and set Instructor user as an Organizer and add some members.
5. Log in as an Instructor user and try to send a group message. Groups > Send Messages

### Proof Screenshots or Video

![image](https://user-images.githubusercontent.com/82648504/129787485-ebfc3b11-1acf-472e-9918-9a5cfc0334c3.png)

### Other information:




### Changelog entry

> Added exclusion in bp-email post type as it causes for email not to work if sender has instructor role from instructor-role plugin
